models/base_sources/arbitrum_base_sources.yml

```diff
@@ -10,14 +10,6 @@ sources:
       - name: transactions
         loaded_at_field: block_time
         columns:
-          - name: block_time
```

Comment:
any reason these are being removed? they appear to be in the base table. columns should likely be all or nothing